cypress/page/payment-step.page.ts

```diff
@@ -7,13 +7,20 @@ class PaymentStep{
         this.paymentButton = ".bankwire"
     }
 
-    public payment(): void {
+    public paymentByBankWire(): void {
         cy.get(this.paymentButton).click()
     }
 
     public proceedToCheckout(): void {
         cy.get(this.proceedToCheckoutButton).click()
     }
+
+    public getConfirmationMessage(): void {
```

Comment:
as this method is called getConfirmationMessage, I would expect to make it to return the title instead of performing the assertion. The assertion part, can be made in the buy-shirt.cy.ts file. When I say assertion part I refer to this code:
```
        .should(
            "have.text",
            "Your order on My Store is complete.",
        );
```